frontend/src/employee-frontend/components/messages/types-view.d.ts

```diff
@@ -9,4 +9,5 @@ export type View = 'RECEIVED' | 'SENT' | 'RECEIVERS' | 'DRAFTS'
 export interface AccountView {
   account: MessageAccount
   view: View
+  selectedUnitId: UUID | undefined
```

Comment:
I'm not entirely sure that the `selectedUnitId` belongs here because, the `AccountView` interface is used to  control the selected message box.